app/code/Magento/Customer/Model/Indexer/Source.php

```diff
@@ -92,7 +92,7 @@ public function count()
      *
      * @return Traversable
      */
-    public function getIterator()
+    public function getIterator(): Traversable
```

Comment:
```suggestion
    #[\ReturnTypeWillChange]
    public function getIterator()
```

Reply:
Hello @ihor-sviziev
This is really true.
This is big BIC IMHO.
We fixed this changes.
@karyna-tsymbal-atwix thank you for the fast fix.